docs/node/migrations/zero-downtime-migrations/index.mdx

```diff
@@ -0,0 +1,135 @@
+
+![Migration Process](../../../../static/img/migration.jpg)
+
+## Migration Proposal
+A network migration is proposed by a validator using a special network governance resolution. The migration proposal includes the activation period, duration, and the chain ID of the new network as shown below.
+
+- `ActivationPeriod`: The amount of blocks before the migration is activated. It starts after the migration is approved via the voting system. The intention is to allow validators to prepare for the migration.
+- `Duration`: The amount of blocks the migration will take to complete.
+- `ChainID`: The new chain ID that the network will migrate to. A new chain ID should always be used for a new network to avoid cross-network replay attacks.
+- `Timestamp`: The time at which the migration was proposed.
+
+The Migration proposal needs to be approved by at least 2/3rd of the validators to start the migration process. The migration proposal will expire if the supermajority of the validators do not approve the proposal within `100800` blocks (roughly 7 days with a 6 second block interval). Once the migration proposal is approved, the network moves into migration mode after the specified activation period.
+
+### Submit Migration Proposal
+
+A validator operator can submit a migration proposal using the `kwil-admin migrate propose` command as shown below. Refer to the [Submit Migration Proposal](/docs/ref/kwil-admin/migrate/propose) documentation for more details.
+
+```bash
+kwil-admin migrate propose --activation-period <activationPeriod> --duration <migrationDuration> --chain-id <newChainID> --rpcserver /tmp/kwil.sock
+```
+
+:::note
+An off-chain communication is required among the validator operators to decide on the details of the migration proposal. One of the validators should submit the proposal to the network for approval. Only one proposal can be applied to the network during the lifetime of the network (per chain-id). Once the proposal is approved, all the other migration proposals will be rejected from the network and no new proposals can be submitted or approved.
+:::
+
+To check if the proposal is submitted successfully, we can get the list of pending migration proposals as shown below:
+
+```bash
+kwil-admin migration list --rpcserver /tmp/kwild.socket
+```
+This list will contain the proposal ID which is used by the validators to vote on the proposal.
+
+
+### Approve Migration Proposal
+
+To start the migration, the migration proposal needs the approval from at least a supermajority of the validators. The validators can submit their vote approval on the migration proposal using the below command.
+
+```bash
+kwil-admin migration approve <proposal-id>
+```
+
+Operators can check the status of the proposal and the votes information as shown below:
+
+```bash
+kwil-admin migration status <proposal-id>
+```
```

Comment:
One annoying aspect of this page is the inconsistency between CLI variables. You switch between `cli command --flag <camelCase>` and `cli command --flag <hyphen-case>`, as well as you do `<snake_case>` in the `genesis.json`.

Everywhere else in the docs, we use `<snake_case>`. The only exception is in [`kwil-admin`](<https://docs.kwil.com/docs/admin/keys#generation>), where Jon added proper CLI doc formatting. IMO, we should try to do everything right (how Jon did it), but if you are looking for a quick fix, switching to snake case is acceptable for now since we do it elsewhere in the docs.

I know that this is minor, but it's one of those things that comes across as sloppy.

Reply:
I made an issue here with more details: https://github.com/kwilteam/docs/issues/153